### ISSUE TITLE
Accessibility - Optional Actions keyboard shortcuts are not announced

### ISSUE THREAD

**Issue by mgreenegit** [wixtoolset/issues#7919]
### WiX Version

4

### .NET or MSBuild or Visual Studio Version

na

### HeatWave Version

na

### Windows Version

11

### Repro Steps

Next - Next - OptionalActions

### Actual Result

All checkboxes should have keyboard shortcuts annouced by narrator

### Expected Result

Only the last checkbox is announced

**Comment by barnson** [comment]
"OptionalActions" isn't a WiX thing.
